scripts/download_validation_reports.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+#User input parameters:
+#$1 filter start date 'yyyy-mm-dd'
+#$2 filter end date 'yyyy-mm-dd'
+
```

Comment:
Can an ACA address be added as an optional additional parameter with a default to localhost?

Reply:
The script now looks for a third argument to use as the ACA address, defaulting to localhost if not found.